pipeline-steps/artifact_build.groovy

```diff
@@ -1,71 +1,106 @@
-def python(Map args) {
+def get_rpc_repo_creds(){
```

Comment:
@hughsaunders can't we just have everything returned? I'm not sure if I'm the right person to review this... So different than usual!